src/_shared/js/messages.js

```diff
@@ -106,11 +106,14 @@ function getWebfonts(fontFamilies) {
 function resizeIframeHeight(height = -1) {
     return height === -1 ?
         timeout(Promise.all(areImagesLoaded().concat(isDocumentLoaded())), 3000)
-        .then(() => read(() => document.body.getBoundingClientRect().height))
+        .then(() => {
+            return read(() => document.body.getBoundingClientRect().height)
+        })
         .then(function(height) {
+            console.log(`*** Sending resize height of ${height}`);
             return sendMessage('resize', { height });
         }) :
-        sendMessage('resize', { height });
+        (console.log(`*** Height !== -1. Sending resize height of ${height}`), sendMessage('resize', { height }));
```

Comment:
let's remove the console logs for the native template we can still use the test one for debugging